modules/seqexec/server/src/test/scala/seqexec/server/SeqTranslateSpec.scala

```diff
@@ -14,7 +14,9 @@ import gem.Observation
 import gem.enum.Site
 import io.chrisdavenport.log4cats.noop.NoOpLogger
 import java.time.LocalDate
+
```

Comment:
why empty lines?

Reply:
That's probably Idea messing with the imports.